### PR TITLE
Handle case when Rtx payload type doesn't exist in offer/answer

### PR DESCRIPTION
**Pull request by zhiyua-git**
*Issue #, if available:* N/A

*Description of changes:* Update code logic to accept case when rtx payload type doesn't exist, it can still continue to check other codecs instead of exiting.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/source/PeerConnection/SessionDescription.c [hunk 1]
```diff
@@ -230,7 +230,7 @@ STATUS setPayloadTypesFromOffer(PHashTable codecTable, PSessionDescription pSess
     return retStatus;
 }
 
-STATUS setTransceiverPayloadTypes(PHashTable codecTable, PDoubleList pTransceievers) {
+STATUS setTransceiverPayloadTypes(PHashTable codecTable, PDoubleList pTransceiver) {
```
Comment (Sean-Der) [inline src/source/PeerConnection/SessionDescription.c]: `pTransceievers` I think is the right name since it is a list of them

**Comment by Sean-Der** [comment]
@zhiyua-git would you mind also getting rid of `RTC_RTX_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE`? We should do a table just for RTX PayloadTypes, I just don't want users to be confused when they see it in the public API.

File: src/source/PeerConnection/SessionDescription.c (hunk 1 above)
Comment (MushMal) [reply src/source/PeerConnection/SessionDescription.c]: +1

Comment (zhiyua-git) [reply src/source/PeerConnection/SessionDescription.c]: hmm, one typo to another typo, will fix it.

**Comment by zhiyua-git** [comment]
> @zhiyua-git would you mind also getting rid of `RTC_RTX_CODEC_H264_PROFILE_42E01F_LEVEL_ASYMMETRY_ALLOWED_PACKETIZATION_MODE`? We should do a table just for RTX PayloadTypes, I just don't want users to be confused when they see it in the public API.

Split into RTX type and RTX table.

**Pull request merged**